backup_and_restore/restore-acs.adoc

```diff
@@ -8,10 +9,53 @@ toc::[]
 [role="_abstract"]
 You can restore {product-title} from an existing backup by using the `roxctl` command-line interface (CLI).
 
-You can use the `roxctl` CLI to restore {product-title} by using the `restore` command. You require either an API token or your administrator password to run this command.
+Depending upon your requirements and the data you have backed up. You can restore from the following types of backups:
 
-include::modules/restore-acs-roxctl-api.adoc[leveloffset=+1]
+. Restore Central database from the Central database backup. Use this to recover from a database failure or data corruption event.
+It allows you to restore and recover the Central database to its earlier functional state.
+. Restore Central from the Central deployment backup. Use this if you are migrating Central to another cluster or namespace. This option restores the configurations of your Central installation.
 
-include::modules/restore-acs-roxctl-admin-pass.adoc[leveloffset=+1]
+[id="restore-central-db-roxctl"]
+== Restoring Central database by using the `roxctl` CLI
+You can use the `roxctl` CLI to restore {product-title} by using the `restore` command. You require an API token or your administrator password to run this command.
 
-include::modules/resume-acs-restore.adoc[leveloffset=+1]
+include::modules/restore-acs-roxctl-api.adoc[leveloffset=+2]
+
+include::modules/restore-acs-roxctl-admin-pass.adoc[leveloffset=+2]
+
+include::modules/resume-acs-restore.adoc[leveloffset=+2]
+
+[id="restore-central-deployment-roxctl"]
+== Restoring Central deployment using the `roxctl` CLI
+You can restore your Central deployment to its original configuration by using the backups you made.
+To successfully restore, you need Central custom resource, `central-tls` and the administrator password.
```

Comment:
This is not necessary for roxctl CLI. It extracts this data from the RHACS backup file.